test/src/sdk-harness/Cargo.toml

```diff
@@ -7,10 +7,10 @@ version = "0.0.0"
 
 [dependencies]
 fuel-core = { version = "0.6", default-features = false }
```

Comment:
Should this be bumped to `0.7` as well?

Reply:
Oh, I guess I missed this one, thanks :)